src/rpc/server.cpp

```diff
@@ -224,11 +224,20 @@ UniValue stop(const JSONRPCRequest& jsonRequest)
     // Accept the deprecated and ignored 'detach' boolean argument
     if (jsonRequest.fHelp || jsonRequest.params.size() > 1)
         throw std::runtime_error(
-            "stop\n"
-            "\nStop Bitcoin server.");
+            "stop ( wait )\n"
+            "\nStop Bitcoin server.\n"
+            "\nArguments:\n"
+            "1. \"wait\"     (int, optional, default=0) Milliseconds to wait before returning to the client\n"
```

Comment:
maybe mention that this is for testing purposes

Reply:
I also don't see reason to use this argument.